runtime/process.go

```diff
@@ -260,8 +260,12 @@ func (p *process) Signal(s os.Signal) error {
 // This should only be called on the process with ID "init"
 func (p *process) Start() error {
 	if p.ID() == InitProcessID {
-		_, err := fmt.Fprintf(p.controlPipe, "%d %d %d\n", 2, 0, 0)
-		return err
+		args := p.container.runtimeArgs
```

Comment:
We're already monitoring the process. Which main that `process.Wait()` gets called by `exitHandler()`.

Add a channel `process.Wait()` to piggy back on it?